client/hosting/performance/site-performance.tsx

```diff
@@ -190,6 +190,21 @@ export const SitePerformance = () => {
 		wpcom_performance_report_url?.hash,
 	] );
 
+	useEffect( () => {
+		const reportTimestamp = performanceReport.performanceReport?.timestamp;
+		const isOlderThan24Hours = reportTimestamp
+			? moment().diff( moment( reportTimestamp ), 'hours' ) > REFRESH_REPORT_INTERVAL
```

Comment:
The units are a bit busted here. `REFRESH_REPORT_INTERVAL` is in milliseconds, but the `'hours'` argument is being passed to the `diff()` method. Either `REFRESH_REPORT_INTERVAL` should be just `24`, or we should drop the optional `'hours'` argument so that `diff()` returns milliseconds.

Reply:
Yup. It should have been 24 instead of the milliseconds